docs/deployment-examples.md

```diff
@@ -0,0 +1,331 @@
+      api_token: 'a1b2c3d4-e5f6-1234-5678-a1b2c3d4e5f6'
+
+    # Example of a second array
+    array02:
+      address: 'array02.fqdn.com'
+      api_token: 'a1b2c3d4-e5f6-1234-5678-a1b2c3d4e5f7'
+    ```
+
+2. **Run the container**
+    Run the container this time specifying a volume (in our case a single file) to pass the token.yaml file from the host server to the guest container.
+    The container expects to see the file here: `/etc/pure-fa-om-exporter/tokens.yaml`
+
+    ```console
+    $ docker run -d -p 9490:9490 --name pure-fa-om-exporter --volume /directorypath/tokens.yaml:/etc/pure-fa-om-exporter/tokens.yaml quay.io/purestorage/pure-fa-om-exporter:latest
+    ```
+3. **Check the container is running**
+    ```console
+    $ docker ps
+    CONTAINER ID   IMAGE                                            COMMAND                  CREATED         STATUS         PORTS                                       NAMES
+    0076ded8a073   quay.io/purestorage/pure-fa-om-exporter:latest   "/pure-fa-om-exporte…"   10 seconds ago   Up 10 seconds   0.0.0.0:9490->9490/tcp, :::9490->9490/tcp   pure-fa-om-exporter
+    ```
+
+4. **Test the exporter**
+    Use `curl` to test the exporter returns results. We don't need to pass the bearer (API) token for authorization as the exporter has a record of these which makes queries simpler.
+    ```console
+    $ curl -X GET 'http://localhost:9490/metrics/array?endpoint=array01 ' -silent | grep ^purefa_info
+    purefa_info{array_name="ARRAY01",os="Purity//FA",system_id="12345678-abcd-abcd-abcd-abcdef123456",version="6.5.0"} 1
+    ```
+
+## Container - TLS - https passing API token with query
+TLS with Docker image is currently not supported. Please use executable binary only for this feature.
+
+# Executable Binary Deployment
+## Binary Specific Prerequisites
+
+Deploying the binary requires [go](https://go.dev) to compile the code and running the binary in a Linux environment.
+1. **Install go**: Go is required: to compile the build into an executable binary
+
+    Clear instructions: can be found here: [https://go.dev/doc/install](https://go.dev/doc/install)
+
+2. **Install git**
+ 
+    ```console
+    $ yum install git
+    ```
+
+3. **Clone git repo**
+ 
+    ```console
+    $ git clone git@github.com:PureStorage-OpenConnect/pure-fa-openmetrics-exporter.git
+    ```
+
+4. **Build the package**
+
+    ```console
+    $ cd pure-fa-openmetrics-exporter
+    $ make build .
+    ```
+
+## Binary - Default - http passing API token with query
+
+5. **Binary - Default - http passing API token with query**
+    ```console
+    $ ls out/bin
+    $ .out/bin/pure-fa-openmetrics-exporter
+    Start Pure FlashArray exporter v1.0.9 on 0.0.0.0:9490
+    ```
+
+6. **Test the exporter**
+    Use `curl` to test the exporter returns results.
+    ```console
+    $ curl -H 'Authorization: Bearer a1b2c3d4-e5f6-1234-5678-a1b2c3d4e5f6' -X GET 'http://localhost:9490/metrics/array?endpoint=array01' -silent | grep ^purefa_info
```

Comment:
maybe placeholders again

Reply:
Resolved in commit 4010cfcac1c820121ac3f1816ef1ed4180d0595b